README.md

```diff
@@ -40,6 +40,7 @@ in the below methods, `keys` is an object of the following form:
 
 ``` js
 {
+  "feedType": "ed25519"
```

Comment:
I've been thinking about this recently with [binary encoding of ids](https://github.com/ssbc/box2-spec/blob/master/encoding/tfk.md).

What I don't like about this is that `ed25519` is a cryptographic curve thing... but that doesn't tell us anything about the feed type... like are the messages in this feed encoded as JSON, or are they some binary format? what's the spec for verifying this `feedType`?

I calling this `feedType` "classic", which happens to be ed25519 curve connection + signing, sha256 hashing, JSON encoding. "gabby" is another `feedType` which uses ed25519 but has some other stuff going on I gather

Reply:
yup, exactly. Same is true for the hash function.

In the protocol meetup last may we therefore renamed the SHA to „scuttlebutt happend anyway“.

Reply:
I was also going to comment about the `feedType` and I might be a little late, but I think this is specifically referring to the type of encryption used in the message.

So, isn't something like `encryption` a possible better name for this property?